### PR TITLE
Replacing logo

### PR DESCRIPTION
**Pull request by mgamini**
tenor-157792109

Signed-off-by: Garrett Amini <kadjar@gmail.com>

### CODE REVIEW

**Comment by cnunciato** [comment]
Hm, when I run this, here's what I see: 

image

File: public/images/habitat-logo-by-chef.svg [hunk 1]
```diff
@@ -1,11 +1,64 @@
...
-    </g>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cnunciato) [inline public/images/habitat-logo-by-chef.svg]: This doesn't seem right. Try opening the file in Sketch, instead, then copying out the SVG code and comparing to the current version; the result should be pretty much just one `path`, similar to line 8 of the original.

File: public/images/image-cat-set.svg [hunk 2]
```diff
@@ -1,134 +1,235 @@
...
-            <use xlink:href="#image-cat-set-k"/>
-          </mask>
-          <path fill="#F3FCFF" d="M44.4294086,0.226358843 L44.4294086,1.08993186 C44.4294086,2.45240062 43.308168,3.55937952 41.9277089,3.55937952 L2.5006943,3.55937952 C1.10843263,3.55937952 -0.000131139247,2.45240062 -0.000131139247,1.08993186 L-0.000131139247,0.226358843 L2.37829767,0.226358843 L2.37829767,0.50601667 C2.37829767,1.15078333 3.36359054,1.67384704 4.59586233,1.67384704 L39.5873105,1.67384704 C40.8073426,1.67384704 41.8053123,1.15078333 41.8053123,0.50601667 L41.8053123,0.226358843 L44.4294086,0.226358843 Z" mask="url(#image-cat-set-l)"/>
-        </g>
-        <path fill="#847264" d="M18.6523722,45.9332371 L26.5862967,45.9332371 C26.5862967,45.1223157 25.9205464,44.4646019 25.0987405,44.4646019 C24.9431219,44.4646019 24.7953717,44.4948119 24.6537413,44.5388321 C24.2061193,43.9031284 23.4629969,43.4853679 22.6193344,43.4853679 C21.7761091,43.4853679 21.0329867,43.9031284 20.5849276,44.5388321 C20.4437343,44.4948119 20.2964212,44.4646019 20.1399284,44.4646019 C19.3185596,44.4646019 18.6523722,45.1223157 18.6523722,45.9332371"/>
-      </g>
-    </g>
-  </g>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cnunciato) [inline public/images/image-cat-set.svg]: Same here -- if I open this in Sketch, delete the marks, then copy out the SVG code, I get a layout that matches what's currently on master.

**Comment by cnunciato** [comment]
Fixes https://github.com/habitat-sh/habitat/issues/4972

**Comment by mgamini** [comment]
@cnunciato manually re-added dimension data to the svgs for now. Rebuilt and tested, looks good.

**Pull request merged**